modules/alb/locals.tf

```diff
@@ -0,0 +1,28 @@
+locals {
+  # aws_lb
+  namespace                     = "devops-ic-alb"
+  access_logs_bucket            = "${local.namespace}-log-bucket"
+  application_target_group_name = "${local.namespace}-application-tg"
+  load_balancer_type            = "application"
+
+  # aws_lb_target_group
+  application_target_group_protocol       = "HTTP"
+  application_target_type                 = "ip"
+  application_target_deregistration_delay = 100
+
+  # health_check
+  health_check_timeout             = 3
```

Comment:
are you going to run rails app? if yes, 3s is small timeout for rails app to start

here you can find more details: https://nimble-co.slack.com/archives/CDZAY1Z4L/p1686799140089159?thread_ts=1686795546.965199&cid=CDZAY1Z4L

Reply:
Thanks for your suggestion, it's really nice to be aware of that. It was updated to a higher threshold in eb4b6d6 ✅